omod/src/main/webapp/resources/htmlforms/vitals.xml

```diff
@@ -0,0 +1,256 @@
+<htmlform formUuid="a000cb34-9ec1-4344-a1c8-f692232f6edd"
+          formName="Vitals"
+          formEncounterType="67a71486-1a54-468f-ac3e-7091a9a79584"
+          formVersion="1.0"
+          formDescription="Client HIV Testing Card"
```

Comment:
@semujju form name - Really?

Reply:
@ssmusoke i didn't have the right name at the time, used that as a place holder.

Reply:
@ssmusoke Name It Vitals